core/controllers/classifier_test.py

```diff
@@ -214,3 +243,4 @@ def test_error_on_invalid_vm_id(self):
         self.post_json(
             '/ml/nextjobhandler', self.payload,
             expect_errors=True, expected_status_int=401)
+
```

Comment:
I think Travis build failed due to this extra line?

Reply:
done.